cukes-rest/src/main/java/lv/ctco/cukesrest/internal/context/BaseContextHandler.java

```diff
@@ -9,10 +9,14 @@
 
     protected List<String> extractGroups(String pattern) {
         List<String> allMatches = new ArrayList<String>();
-        Matcher m = Pattern.compile(GROUP_PATTERN).matcher(pattern);
-        while (m.find()) {
-            allMatches.add(m.group(1));
+
+        if (pattern != null) {
```

Comment:
What was the problem with pattern != null here? Maybe there is a better way to fix the problem?